kubernetes-client/src/main/java/io/fabric8/kubernetes/client/utils/TokenRefreshInterceptor.java

```diff
@@ -50,10 +50,11 @@ public Response intercept(Chain chain) throws IOException {
       }
       AuthInfo currentAuthInfo = KubeConfigUtils.getUserAuthInfo(kubeConfig, currentContext);
       // Check if AuthProvider is set or not
-      if (currentAuthInfo != null && currentAuthInfo.getAuthProvider() != null) {
+      if (currentAuthInfo != null) {
         response.close();
         String newAccessToken;
-        if (currentAuthInfo.getAuthProvider().getName().toLowerCase().equals("oidc")) {
+        // Check if AuthProvider is set to oicd
```

Comment:
oicd-> oidc

Reply:
@skabashnyuk : Thanks for noticing. Could you please submit a quick PR to fix this?